po/zh-CN.po

```diff
@@ -4101,118 +3917,108 @@ msgstr "用一个值（例如 `break 8`）来中断 `loop` 并将其输出。"
-"std/iter/trait.FromIterator.html), which [`HashSet`](https://doc.rust-lang."
-"org/std/collections/struct.HashSet.html#impl-FromIterator%3CT%3E-for-"
-"HashSet%3CT,+S%3E) implements."
+"[`collect`](https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method."
+"collect) relies on [`FromIterator`](https://doc.rust-lang.org/std/iter/trait."
+"FromIterator.html), which [`HashSet`](https://doc.rust-lang.org/std/collections/struct."
+"HashSet.html#impl-FromIterator%3CT%3E-for-HashSet%3CT,+S%3E) implements."
 msgstr ""
-"[`collect`](https://doc.rust-lang.org/stable/std/iter/trait.Iterator."
-"html#method.collect) 依赖 [`HashSet`](https://doc.rust-lang.org/std/iter/"
-"trait.FromIterator.html) 实现的 `FromIterator`。"
+"[`collect`](https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method."
+"collect) 依赖 [`HashSet`](https://doc.rust-lang.org/std/iter/trait.FromIterator.html) 实"
+"现的 `FromIterator`。"
 
 #: src/basic-syntax/static-and-const.md:1
 msgid "Static and Constant Variables"
 msgstr "静态 (Static) 变量和常数 (Constant) 变量"
 
 #: src/basic-syntax/static-and-const.md:3
 msgid ""
-"Static and constant variables are two different ways to create globally-"
-"scoped values that cannot be moved or reallocated during the execution of "
-"the program. "
+"Static and constant variables are two different ways to create globally-scoped values "
+"that cannot be moved or reallocated during the execution of the program. "
 msgstr ""
-"静态变量和常量变量是创建全局范围值的两种不同方法，这类值在程序执行期间无法移"
-"动或重新分配。"
+"静态变量和常量是创建全局范围值的两种不同方法，这类值在程序执行期间无法移动或重新分配。"
 
 #: src/basic-syntax/static-and-const.md:6
 msgid "`const`"
 msgstr "`const`"
 
 #: src/basic-syntax/static-and-const.md:8
 msgid ""
-"Constant variables are evaluated at compile time and their values are "
-"inlined wherever they are used:"
+"Constant variables are evaluated at compile time and their values are inlined wherever "
+"they are used:"
 msgstr "系统会在编译时对常量变量进行求值；无论在何处使用，其值都会被内嵌："
 
 #: src/basic-syntax/static-and-const.md:30
 msgid ""
-"According to the [Rust RFC Book](https://rust-lang.github.io/rfcs/0246-const-"
-"vs-static.html) these are inlined upon use."
+"According to the [Rust RFC Book](https://rust-lang.github.io/rfcs/0246-const-vs-static."
+"html) these are inlined upon use."
 msgstr ""
-"根据 [Rust RFC Book](https://rust-lang.github.io/rfcs/0246-const-vs-static."
-"html) 这些变量在使用时是内联 (inlined) 的。"
+"根据 [Rust RFC Book](https://rust-lang.github.io/rfcs/0246-const-vs-static.html) 这些变量"
+"在使用时是内联 (inlined) 的。"
 
 #: src/basic-syntax/static-and-const.md:32
 msgid ""
-"Only functions marked `const` can be called at compile time to generate "
-"`const` values. `const` functions can however be called at runtime."
+"Only functions marked `const` can be called at compile time to generate `const` values. "
+"`const` functions can however be called at runtime."
 msgstr ""
-"在编译时只能调用标记为“const”的函数以生成“const”值。不过，可在运行时调"
-"用“const”函数。"
+"在编译时只能调用标记为“const”的函数以生成“const”值。不过，可在运行时调用“const”函数。"
```

Comment:
I noticed sometimes \`\` is changed to "", while sometimes we keep using \`\` in translated version. Is there a guideline for it?

Reply:
I do not exactly know because this content is not translated by me.😂 I will review them in the future.